<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatricipantClass</class>
 <widget class="QMainWindow" name="PatricipantClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patricipant</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidgetQuizes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAnswer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>233</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAnswer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScore">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
